doc/manuals/python/process.ipynb

```diff
@@ -143,175 +144,174 @@
     "\n",
     "\\# -- end of file --"
    ],
-   "cell_type": "markdown",
    "metadata": {}
   },
   {
+   "cell_type": "markdown",
    "source": [
     "Next we will detail the differentiation between inheriting from `process.KwiverProcess` and `process.PythonProcess`"
    ],
-   "cell_type": "markdown",
    "metadata": {}
   },
   {
+   "cell_type": "markdown",
    "source": [
     "Pure python process implementing functionality from python\n",
     "\n",
     "For this example we will be implementing a *very* simple 3d point class"
    ],
-   "cell_type": "markdown",
    "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
-    "import numpy as np\n",
-    "\n",
-    "class pt3d(object):\n",
-    "    def __init__(self, x = 0, y = 0, z = 0, dtype = np.float32):\n",
-    "        self.x = x\n",
-    "        self.y = y\n",
-    "        self.z = z\n",
-    "        self.dtype = dtype\n",
-    "    def as_array(self):\n",
-    "        return np.ndarray((1,3), dtype=self.dtype, buffer=[self.x,self.y,self.z])\n",
-    "    def as_mat(self):\n",
-    "        return np.matrix(self.as_array())\n",
-    "    def __str__(self):\n",
+    "import numpy as np\r\n",
+    "\r\n",
+    "class pt3d(object):\r\n",
+    "    def __init__(self, x = 0, y = 0, z = 0, dtype = np.float32):\r\n",
+    "        self.x = x\r\n",
+    "        self.y = y\r\n",
+    "        self.z = z\r\n",
+    "        self.dtype = dtype\r\n",
+    "    def as_array(self):\r\n",
+    "        return np.ndarray((1,3), dtype=self.dtype, buffer=[self.x,self.y,self.z])\r\n",
+    "    def as_mat(self):\r\n",
+    "        return np.matrix(self.as_array())\r\n",
+    "    def __str__(self):\r\n",
     "        return \"(%s)\" % \",\".join(str(self.x), str(self.y), str(self.z))"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
+   "cell_type": "markdown",
    "source": [
-    "Now that we have a basic data type, we need to create processes that can handle this datatype. Preexisting Kwiver processes will not support it as this data type is outside of vital. For demonstration purposes, the KwiverProcess utility mixin will leveraged to improve clarity and readability."
+    "Now that we have a basic data type, we need to create processes that can handle this datatype. Preexisting Kwiver processes will not support it as this data type is outside of vital. For demonstration purposes, the KwiverProcess utility mixin will used.\r\n"
```

Comment:
KwiverProcess utility mixin will used. ... still needs some cleanup ... "will be used'?